DataManagementSystem/DB/FileCatalogComponents/FileManagerBase.py

```diff
@@ -793,6 +793,8 @@ def _addReplicas( self, lfns, connection = False ):
     connection = self._getConnection( connection )
     successful = {}
     res = self._findFiles( lfns.keys(), ['DirID', 'FileID', 'Size'], connection = connection )
+    if not res['OK']:
+      return S_ERROR( "Failed to run _findFiles" )
```

Comment:
The fix is a good idea, but please return the original error, otherwise we have no way to track down what happens :-)